database/PostgreSQL.js

```diff
@@ -1,35 +1,82 @@
 /* eslint-disable no-console */
-const { Sequelize, DataTypes } = require('sequelize');
+// postgres DB
+/* eslint-disable no-unused-vars */
+/* eslint-disable prefer-const */
+const Promise = require('bluebird');
 
+const initOptions = {
+  promiseLib: Promise,
+};
 
-const sequelize = new Sequelize('item_details', 'oso', 'hrr45', {
+const pgp = require('pg-promise')(initOptions);
+
+const connection = {
+  user: 'oso',
+  password: 'hrr45',
```

Comment:
by the time you deploy you'll probably want to swap this out for an environment variable